README.md

```diff
@@ -6,13 +6,13 @@ Learn how Remix can supercharge your React development.
 
 The code for each chapter can be found in the corresponding folder. For example, `2-creating-a-new-remix-app` contains the code for _Chapter 2, Creating a new Remix App_.
 
-Each chapter provides instructions for how to get started and how to incrementally reach the final code. The best starting point is the `README.md` file in each folder. The structure of each chapter may vary, but usually you will find a `start` and a `solution` folder. The `start` folder contains the code that you will start with. The `solution` folder contains the code that you will end up with.
+Each chapter provides instructions for how to get started and how to incrementally reach the final code. The best starting point is the `README.md` file in each folder. The structure of each chapter may vary, but starting in [chapter 4](4-routing-in-remix/) you will typically find a `start` and a `solution` folder. The `start` folder contains the code that you will start with. The `solution` folder contains the code that you will end up with.
```

Comment:
I thought it might be a little confusing to tell people there will be a `start` and `solution` folder and then there's not for the first 2 chapters.